Controllers/LoginController.php

```diff
@@ -24,22 +24,18 @@ public function index()
 
     public function index_action()
     {
-        if (!empty($_POST['email']) && !empty($_POST['password'])) {
-            $email = $_POST['email'];
-            $password = $_POST['password'];
-
-            $u = new Users();
-            if ($u->validateLogin($email, $password)) {
-                header("Location: " . BASE_URL);
-                exit;
-            } else {
-                $_SESSION['errorMsg'] = 'Usuário e/ou senha errados!';
-            }
-        } else {
-            $_SESSION['errorMsg'] = 'Preencha os campos abaixo.';
-        }
+        $email = (filter_input(INPUT_POST, 'email', FILTER_VALIDATE_EMAIL) );
+        $password = (filter_input(INPUT_POST, 'password', FILTER_UNSAFE_RAW) );
```

Comment:
Não há necessidade dos parênteses de fora.
```suggestion
        $email = filter_input(INPUT_POST, 'email', FILTER_VALIDATE_EMAIL);
        $password = filter_input(INPUT_POST, 'password', FILTER_UNSAFE_RAW);
```